### PR TITLE
Docstrings for `gss.logistics`

### PR DESCRIPTION
**Pull request by bharat-thotakura**
Mainly class docstrings for logistics.py

### CODE REVIEW

File: general-superstaq/general_superstaq/logistics.py [hunk 1]
```diff
@@ -12,7 +12,9 @@
 
 
 @dataclass
-class TSPOutput:  # pylint: disable=missing-class-docstring
+class TSPOutput:
+    """A class to store data from the /tsp endpoint."""
```
Comment (richrines1) [inline general-superstaq/general_superstaq/logistics.py:16]: nit
```suggestion
    """A class to store data returned from the /tsp endpoint."""
```

File: general-superstaq/general_superstaq/logistics.py [hunk 2]
```diff
@@ -36,7 +38,9 @@ def read_json_tsp(json_dict: gss.TSPJson) -> TSPOutput:
 
 
 @dataclass
-class WarehouseOutput:  # pylint: disable=missing-class-docstring
+class WarehouseOutput:
+    """A class to store data from the /warehouse endpoint."""
```
Comment (richrines1) [inline general-superstaq/general_superstaq/logistics.py:42]:
```suggestion
    """A class to store data returned from the /warehouse endpoint."""
```

File: general-superstaq/general_superstaq/logistics.py [hunk 3]
```diff
@@ -45,9 +49,9 @@ class WarehouseOutput:  # pylint: disable=missing-class-docstring
 
 
 def read_json_warehouse(json_dict: gss.WareHouseJson) -> WarehouseOutput:
-    """Reads out returned JSON from SuperstaQ API's warehouse endpoint.
+    """Reads out returned JSON from Superstaq API's /warehouse endpoint.
     Args:
-        json_dict: a JSON dictionary matching the format returned by /warehouse endpoint
+        json_dict: a JSON dictionary matching the format returned by the /warehouse endpoint.
```
Comment (dowusu-antwi) [inline general-superstaq/general_superstaq/logistics.py:54]:
```suggestion
        json_dict: A JSON dictionary matching the format returned by the /warehouse endpoint.
```

**Review by richrines1 — APPROVED** [review]
two tiny nits then lgtm!

**Review by dowusu-antwi — COMMENTED** [review]
Capitalizing for some args:

File: general-superstaq/general_superstaq/logistics.py (hunk 3 above)
Comment (bharat-thotakura) [reply general-superstaq/general_superstaq/logistics.py]: Done.

**Pull request merged**